modules/ROOT/pages/migration/upgrading_5.0.x_7.0.0.adoc

```diff
@@ -0,0 +1,243 @@
+* Deployment examples based on one of the `Ubuntu with Docker Compose` deployments have been updated/upgraded.
+
+=== How to Identify if You Are Affected
+
+* If you are using removed or deprecated environment variables in your config.
+** See the xref:deployment/services/env-var-changes.adoc[Added and Removed Environment Variables] documentation.
+
+* Because the xref:{s-path}/activitylog.adoc[activitylog] service has been added and is started automatically, _everyone_ who upgrades is affected.
+
+* The xref:{s-path}/collaboration.adoc[collaboration] service has been added. Though this service needs to be started manually, it requires a secret (preferred a random UUID string) when started. For safety reasons on a fresh install, this secret is added when running the `ocis init` command and should be added on upgrades. The used upgrade procedure provides a secret if not present in the config. Note that the secret can be overwritten in the config or with an environment variable if necessary.
+
+** If you have used web office document servers such as Collabora, ONLYOFFICE or Microsoft using the WOPI protocol with the external https://github.com/cs3org/wopiserver[cs3org/wopi] server, we highly recommend switching to the xref:{s-path}/collaboration.adoc[collaboration] service which requires deployment reconfiguration which is a breaking change. Though the environment variables from the xref:{s-path}/app-provider.adoc[app-provider] service formerly used for web office document deployments are currently not deprecated, the are a candidate for future deprecations.
+
+* If you have used a rolling release and deployed web office document servers like the xref:depl-examples/ubuntu-compose/ubuntu-compose-prod.adoc[Local Production Setup] or the xref:depl-examples/ubuntu-compose/ubuntu-compose-hetzner.adoc[Deployment on Hetzner] deployment example, you should follow the reconfiguration section below because the deployment files have been updated/upgraded. 
+
+=== How to Manage the Change
+
+==== Manage deprecated variables
+
+* Environment variables that have been deprecated without successor can safely be removed from the configuration. These envvars do not harm as they are not used anymore.
+* Environment variables that have been deprecated with a successor need to be updated accordingly.
+* Environment variables that habe been removed from Infinite Scale can safely be removed from the config if set manually.
+
+==== Add new Infnite Scale config settings
+
+To do so, the following steps need to be taken, here is the *overview*:
+
+. Get the config diff via command
+. Apply the diff created
+. Start Infinite Scale as usual
+
+See the next section for *details* how to update the config settings.
+
+=== Update Config Settings
+
+Run the `ocis init --diff` command:
+
+Because this will now run the new ocis code, it will create two files to help manage the needed changes. Note that the patch file is only created if changes are found:
```

Comment:
```suggestion
Because this will now run the new ocis code, it will create two files in the config directory to help manage the needed changes. Note that the patch file is only created if changes are found:
```